services/motion/builtin/builtin.go

```diff
@@ -38,7 +38,10 @@ func init() {
 			) (motion.Service, error) {
 				return NewBuiltIn(ctx, deps, conf, logger)
 			},
-			WeakDependencies: []internal.ResourceMatcher{internal.SLAMDependencyWildcardMatcher},
+			WeakDependencies: []internal.ResourceMatcher{
+				internal.SLAMDependencyWildcardMatcher,
+				internal.ComponentDependencyWildcardMatcher, // TODO(rb): when a better matcher system is built make this match Actuators
```

Comment:
I think we actually just want everything here.

If we had actuator -> static component with a geometry -> actuator, we'd need that middle thing to exist, no?

Reply:
Thats true.  So your suggestion is remove the comment?

Reply:
Yes. We want this to remain.